ECLIPSE_README.md

```diff
@@ -0,0 +1,39 @@
+# Working with Atlas in Eclipse
+
+## First time setup
+1. Clone the Atlas repository from your fork into the desired local location.
+
+2. Import the project into Eclipse using "File" -> "Import" -> "Gradle" -> "Existing Gradle Project"
+
+3. From the command line, run
+```
+$ cd /path/to/local/Atlas/repo
+$ ./gradlew clean build -x test -x integrationTest
+```
+
+4. Back in Eclipse, right click the Atlas project in Package Explorer and click "Refresh"
+  1. NOTE: You should now see a "gen" folder appear in the package explorer
```

Comment:
The problem I had with that is for some reason it just goes ahead and deletes the generated code folder, and I can't figure out why. It may have something to do with the source set config. I am currently playing around with that on a new branch on my local machine.